_data/sb18281/sb18281.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Simon Barnes # No longer than 18 characters
+institution: Massey University # no longer than 58 characters
+profile_pic: Simon.jpg # Relative path to the image you uploaded
+quote: "Progress isn't always improvement, it's just making a step towards your end goal" # No longer than 100 characters
```

Comment:
Remove double quotes from begin and end.

Reply:
Hey Mohammed,

Sorry about that! I've removed the quotes, and fixed up the task list (I didn't even notice those extra spaces sneak in there!).

Let me know if there's anything else needed.

Cheers,

Simon